src/lib/rest/rest.cpp

```diff
@@ -193,6 +195,23 @@ static int uriArgumentGet(void* cbDataP, MHD_ValueKind kind, const char* ckey, c
       ciP->uriParamTypes.push_back(val);
     }
   }
+  else if (key == URI_PARAM_Q)
```

Comment:
The commit 94f0b76 implements passing of the string filter to mongoQueryContext.
Before, ewhat happended was that the 'qString' was parsed upon request arrival, and then not used. Instead of using the parsed qString (stringFilterP), que original string was added as a scope and then the parsing was done again. It worked, of course, but not the way it was meant.